pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1084,6 +1084,10 @@ impl Timeline {
         let mut result = HashMap::new();
         for (k, v) in kv {
             let v = v?;
+            if v.is_empty() {
+                // This is a tombstone -- we can skip it.
```

Comment:
Could you elaborate on this comment: who places the tombstones (a rust doc reference to a function would be ideal).